tf/backends/nonprod.tf

```diff
@@ -0,0 +1,4 @@
+resource_group_name  = "gratibot-azure-data"
+storage_account_name = "gratibotazuredatatfnp"
+container_name       = "tfstate"
+key                  = "terraform.tfstate"
```

Comment:
Should this be `nonprod.terraform.tfstate`?

Reply:
Same as above. These resources were created as a result of `aws-terraform`: https://github.com/liatrio/azure-terraform/blob/main/users-and-groups/gratibot.tf

Reply:
Same as above. These resources were created as a result of `aws-terraform`: https://github.com/liatrio/azure-terraform/blob/main/users-and-groups/gratibot.tf